### PR TITLE
State features available to EE and Aura

### PR DESCRIPTION
**Pull request by ali-ince**

### CODE REVIEW

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-dataflow-connector-14.surge.sh

File: common-content-dataflow/modules/ROOT/partials/job-specification.adoc [hunk 1]
```diff
@@ -158,6 +158,10 @@ Things to pay attention to:
 ]
 ----
 
+[IMPORTANT]
+====
+`keys`, `unique` and `mandatory` options require Aura or Neo4j Enterprise Edition, and will not have any effect when run against a Neo4j Community Edition installation.
```
Comment (fbiville) [inline common-content-dataflow/modules/ROOT/partials/job-specification.adoc:163]: That won't work with Aura Free, will it? Is Aura Free EE as well?

File: common-content-dataflow/modules/ROOT/partials/prerequisites.adoc [hunk 2]
```diff
@@ -98,6 +98,12 @@ To log into a server having a custom authentication scheme.
 =====
 ====
 
+[IMPORTANT]
+====
+Dataflow Flex Templates for BigQuery and Google Cloud to Neo4j are designed to make use of link:https://neo4j.com/docs/cypher-manual/current/constraints/[Constraints] which is only available in Aura or Neo4j Enterprise Edition installations.
```
Comment (fbiville) [inline common-content-dataflow/modules/ROOT/partials/prerequisites.adoc:103]:
```suggestion
Dataflow Flex Templates for BigQuery and Google Cloud to Neo4j are designed to make use of link:https://neo4j.com/docs/cypher-manual/current/constraints/[Constraints] which are only available in Aura or Neo4j Enterprise Edition installations.
```

**Review by fbiville — APPROVED** [review]

**Review by stefano-ottolenghi — APPROVED** [review]

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-dataflow-connector-14.surge.sh

**Pull request merged**